README.md

```diff
@@ -2,13 +2,12 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+* This is **an address book application targeted for private school teachers and educators**.<br>
```

Comment:
I don't think it will be considered as address book anymore. Maybe we can discuss it in meeting.

Reply:
Added to things to discuss in the collab notes